src/components/checkbox/checkbox.bootstrap.scss

```diff
@@ -2,88 +2,93 @@
 
 @mixin theme() {
     igc-checkbox::part(control) {
-        $size: var(--size, utils.rem(14px));
+        $size: var(--size, utils.rem(16px));
         width: $size;
         height: $size;
         min-width: $size;
+        transition: transparent;
```

Comment:
The transition should be:  transition: background-color .15s ease-in-out, box-shadow .15s ease-in-out;